redfish-core/lib/network_protocol.hpp

```diff
@@ -32,6 +32,7 @@
 #include <optional>
 #include <string_view>
 #include <variant>
+#include <vector>
```

Comment:
why do we need this, previously we had vectors working with this explicit definition. Its already included in one of the headers above.

Reply:
See upstream discussion at https://gerrit.openbmc.org/c/openbmc/bmcweb/+/71026